impl/src/planning.rs

```diff
@@ -189,16 +181,11 @@ impl CrateCatalogEntry {
 
   /** Yields the expected location of the build file (relative to execution path). */
   pub fn local_build_path(&self, settings: &RazeSettings) -> String {
-    match settings.genmode {
-      GenMode::Remote => format!(
-        "remote/{}.{}",
-        &self.package_ident, settings.output_buildfile_suffix,
-      ),
-      GenMode::Vendored => format!(
-        "vendor/{}/{}",
-        &self.package_ident, settings.output_buildfile_suffix,
-      ),
-    }
+    let prefix = match settings.genmode {
+      GenMode::Remote => "remote",
+      GenMode::Vendored => "vendor",
+    };
+    format!("{}/{}.{}", prefix, &self.package_ident, settings.output_buildfile_suffix)
```

Comment:
I think this is a regression. In my repo this is generating e.g. `tokio-io-0.1.12.BUILD.bazel`. The previous code used a `.` for remote and `/` for vendor

Reply:
Thanks for the catch